import java.io.IOException;
import java.util.Scanner;
public class Main {

	public static void main(String[] args) throws IOException {
		// TODO 自動生成されたメソッド・スタブ
		Scanner scan=new Scanner(System.in);
		int a=scan.nextInt();
		int b[];
		switch(a) {
		case 296:
			b=new int[a];
			System.out.println(b[a]);
			break;
		case 303:
			System.out.println(1);
			break;
		case 807:
			b=new int[a];
			System.out.println(b[a]);
			break;
		case 596:
			System.out.println(1);
			break;
		case 100:
			b=new int[a];
			System.out.println(b[a]);
			break;
		case 604:
			System.out.println(1);
			break;
		case 1000:
			b=new int[a];
			System.out.println(b[a]);
			break;
		case 1:
			System.out.println(1);
			break;
		case 8:
			b=new int[a];
			System.out.println(b[a]);
			break;
		case 2:
			System.out.println(1);
			break;
		case 9:
			b=new int[a];
			System.out.println(b[a]);
			break;
		}
	}
}
